torchgeo/transforms/transforms.py

```diff
@@ -69,3 +74,147 @@ def forward(self, sample: Dict[str, Tensor]) -> Dict[str, Tensor]:
             sample["boxes"] = sample["boxes"].to(boxes_dtype)
 
         return sample
+
+
+class _ExtractTensorPatches(GeometricAugmentationBase2D):
+    """Chop up a tensor into a grid."""
+
+    def __init__(self, window_size: Union[int, Tuple[int, int]]) -> None:
+        """Initialize a new _ExtractTensorPatches instance.
+
+        Args:
+            window_size: the size of each patch
+        """
+        super().__init__(p=1)
+        self.flags = {"window_size": window_size}
+
+    def compute_transformation(
+        self, input: Tensor, params: Dict[str, Tensor], flags: Dict[str, Any]
+    ) -> Tensor:
+        """Compute the transformation.
+
+        Args:
+            input: the input tensor
+            params: generated parameters
+            flags: static parameters
+
+        Returns:
+            the transformation
+        """
+        out: Tensor = self.identity_matrix(input)
```

Comment:
This isn't correct but something is required and we don't actually use it. I'll iron out the details when we upstream this.